docs/configuration.rst

```diff
@@ -61,4 +84,28 @@ For other use cases, you can customize these configuration options in your ``con
 
    Type: bool
 
-   Notes: If this option is set to ``True``, the extension omits any prefix values from the URLs, including explicit values for ``notfound_default_language`` and ``notfound_default_version``.
+   .. note::
+
+      If this option is set to ``True``, the extension omits any prefix values from the URLs,
+      including explicit values for ``notfound_default_language`` and ``notfound_default_version``.
+
+   .. deprecated:: 0.5
+
+      ``notfound_no_urls_prefix`` is deprecated.  Use :confval:`notfound_urls_prefix` instead
+
+.. confval:: notfound_urls_prefix
+
+   Prefix added to all the URLs generated in the 404 page.
+
+   Default: ``'/en/latest/'``
+
+   Type: string
+
+   .. warning::
+
+      Make sure this config starts and ends with a ``/``.
+      Otherwise, you may have unexpected behaviours.
```

Comment:
Shouldn't the code just validate this?

Reply:
It already validates this and raise a `UserWarning` warning.